zephyr_static_export.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+MAKE_FOLDER=./tensorflow/lite/micro/tools/make
+DL_FOLDER=$MAKE_FOLDER/downloads
+
+# Download files
+make -C $MAKE_FOLDER TENSORFLOW_ROOT=`pwd`/
+
+# Copy the files we need
+mkdir -p third_party_static/flatbuffers
+mkdir -p third_party_static/gemmlowp
+mkdir -p third_party_static/ruy/ruy
+mkdir -p third_party_static/kissfft
+
+cp $DL_FOLDER/flatbuffers/LICENSE third_party_static/flatbuffers/LICENSE
+cp -r $DL_FOLDER/flatbuffers/include third_party_static/flatbuffers/include
```

Comment:
Makes it go in `third_party_static/flatbuffers/include/include` (duplicated include)

```suggestion
cp -r $DL_FOLDER/flatbuffers/include third_party_static/flatbuffers
```